src/workerd/server/server-test.c++

```diff
@@ -1013,6 +1013,7 @@ KJ_TEST("Server: capability bindings") {
                 `    items.push(await (await env.r2.get("baz")).text());
                 `    await env.queue.send("hello");
                 `    items.push("Hello from Queue\n");
+                `    items.push(`Hello from Hyperdrive(${env.hyperdrive.user})\n`);
```

Comment:
Could we have a test that actually calls `connect()`? Or is that too difficult?

Have you at least confirmed locally that it's able to successfully connect to a real DB?